spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDefinitionController.java

```diff
@@ -16,10 +16,19 @@
 
 package org.springframework.cloud.dataflow.server.controller;
 
+import java.util.ArrayList;
```

Comment:
Why? this is an old standard and not fitting neither the Boot nor Spring Framework standards. Other projects have adopted SF code conventions as well. How do we plan to provide a reference to contributors?